### DISCUSSION TITLE
VSCode and Pyenv windows

### DISCUSSION THREAD

**Discussion by nuffink-sudo** [microsoft/vscode-discussions#2006] [Extension Development QnA]
Hi new to this forum so hoping this question is in the right place 

Unable to get vscode to work with pyenv-win - have got all the pyenv bin and shims path etc in the User and main Path variables (paranoid entry) along with the PYENV_ROOT etc and other PYENV variables.
Pyenv is running fine in powershell and have several python versions in there and able to create all the normal virtual envs too so would take it its all fine there but when selecting interpreter in vscode it doesnt recognise any pyenv shims nor takes the full path of pyenv to establish any versions - have searched Google - can see on SO its still outstanding from like a posting in 2021 - anyone found a solution to this or does it just not work in which case i will have to install a normal python environment?
Thanks for any help and hopefully this is not addressed in a topic already (i did search)

**Comment by gjsjohnmurray** [comment]
> Hi new to this forum so hoping this question is in the right place

Please read this pinned thread for guidance about where to go.

https://github.com/microsoft/vscode-discussions/discussions/5

**Reply by nuffink-sudo** [reply]
OK I had read that and obvs misread that this is a forum for extensions creation etc in which apologies - please remove the post as its not relevant and will search elsewhere - thanks
